components/app-framework/src/widgets/log-viewer/log-viewer.directive.js

```diff
@@ -510,6 +535,9 @@
       var onResize = _.debounce(function () {
         resizeAllDivs();
         paused = false;
+        if (deferredAppend) {
```

Comment:
I noticed that if we get an event during a resize we won't update the log area This will ensure we update following any resize.